packages/ckeditor5-watchdog/src/editorwatchdog.ts

```diff
@@ -165,15 +172,21 @@ export default class EditorWatchdog<TEditor extends Editor = Editor> extends Wat
 				console.error( 'An error happened during the editor destroying.', err );
 			} )
 			.then( () => {
-				if ( typeof this._elementOrData === 'string' ) {
-					return this.create( this._data, this._config, this._config!.context );
-				} else {
-					const updatedConfig = Object.assign( {}, this._config, {
-						initialData: this._data
-					} );
+				// if ( typeof this._elementOrData === 'string' ) {
+				// 	return this.create( this._data, this._config, this._config!.context );
+				// } else {
+				// 	const updatedConfig = Object.assign( {}, this._config, {
+				// 		initialData: this._data
+				// 	} );
+				//
+				// 	return this.create( this._elementOrData, updatedConfig, updatedConfig.context );
+				// }
 
-					return this.create( this._elementOrData, updatedConfig, updatedConfig.context );
-				}
+				EditorWatchdogInitPlugin.editorData = this._data!;
```

Comment:
You can pass this in config (define internal/temp config property just for this plugin).